tekton/README.md

```diff
@@ -88,6 +111,10 @@ image registry instead.
 - [`publish-run.yaml`](publish-run.yaml) - This example `TaskRun` and
   `PipelineResources` demonstrate how to invoke `publish.yaml` (see
   [Creating a new release](#creating-a-new-release))
+  
+- You can use [`tkn`](https://github.com/tkn/cli) to run the [release
+  pipeline](./release-pipeline.yaml) (see [Creating a new
+  release](#creating-a-new-release))
```

Comment:
🎉

Reply:
Whoops I think https://github.com/tkn/cli is wrong?

Reply:
Lol 😂